### DISCUSSION TITLE
Use case?

### DISCUSSION THREAD

**Discussion by AndyClifton** [typst/typst#202] [General]
This looks really interesting, but I'm curious what the use case is that motivates the development of a new typesetting engine. Some questions:

- Who should be using this, and for what purpose? 
- What gap is it designed to fill, or what challenge is it designed to solve? 
- Does it replace or complement LaTeX for typically LaTeX tasks, like journal papers or technical reports, or should I think of it more like a simpler version of the LaTeX standalone package?

**Comment by AndyClifton** [comment]
Ah, OK, I see this is somewhat addressed on the App homepage at [typst.app](https://typst.app/). I'm still interested in hearing what the developers are targeting though.

**Comment by Internal-Tide** [comment]
Howerer, It's an awkward situation，for most people, the don't need to write a book, if writing the papers, editors seems more like latex.

**Comment by LectronPusher** [comment]
You might be interested by the creator's original Master's Thesis: <https://www.user.tu-berlin.de/laurmaedje/programmable-markup-language-for-typesetting.pdf>

Section 2 goes in-depth on the history and some limitations of other systems.
